pkgs/by-name/re/retrospy/package.nix

```diff
@@ -57,18 +65,13 @@ buildDotnetModule {
 
   inherit executables;
 
-  postInstall = ''
-    mkdir -p $out/share/retrospy
-    ${builtins.concatStringsSep "\n" (map (e: "cp ./${e}.ico $out/share/retrospy/${e}.ico") executables)}
-  '';
```

Comment:
Use something like convert from imagemagick to convert to png and install at `$out/share/icons/hicolor/"$size"x"$size"/apps/$e.png`